src/NuGet.Clients/NuGet.SolutionRestoreManager/AuditCheckResultCachingService.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System.ComponentModel.Composition;
+using NuGet.PackageManagement;
+
+namespace NuGet.SolutionRestoreManager
+{
+    [Export(typeof(IAuditCheckResultCachingService))]
```

Comment:
```suggestion
    /// <Summary>Used for packages.config NuGetAudit, not PackageReference</Summary>
    [Export(typeof(IAuditCheckResultCachingService))]
```

Or something similar. Once this PR is merged, people in the team aren't going to have the context for the design when reading code and stumbling onto this.